src/yaml/yaml.cr

```diff
@@ -1,5 +1,38 @@
 require "./*"
 
+# The YAML module serves to deserialize data from YAML into accessible structure.
```

Comment:
Do you say HTML data or CSS data? Granted some people say XML data, but I think it's kind of redundant.